### PR TITLE
Adding a test SQL database for the iris dataset.

### PR DESCRIPTION
**Pull request by tannergooding**
CC. @eerhardt, @ericstj

### CODE REVIEW

**Comment by eerhardt** [comment]
Why have an `iris.zip` file inside of a `.nupkg`, which is already a .zip file? Why not just have the contents of the .zip file directly in the package? That way your tests don't need to unzip the .zip file.

**Comment by tannergooding** [comment]
It's the same setup that `TestModels` is using, it will also make it easier to only grab the databases that you want when/if this expands in the future.

**Comment by eerhardt** [comment]
> It's the same setup that TestModels is using

The difference is that `TestModels` contains ML.NET models, which ARE .zip files. That's what ML.NET loads - .zip files.

Here you are taking the files that need to be loaded - the .mdf/ldf files - and putting them in .zip files, which is NOT what gets consumed. So your tests need to do the unzipping.

>  it will also make it easier to only grab the databases that you want when/if this expands in the future.

I don't see how that is the case. Either way, you are referencing the NuGet package. You will still get them all.

**Comment by ericstj** [comment]
Unrelated to this change, but it might be nice to do a similar conversion to what was done for corefx-testdata: https://github.com/dotnet/corefx-testdata/commit/6e958b1164e5648f3ac7c28a9c315d91e8a8e51c

File: Microsoft.ML.TestDatabases.nuspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.ML.TestDatabases</id>
+    <version>0.0.5-test</version>
```
Comment (ericstj) [inline Microsoft.ML.TestDatabases.nuspec:5]: Starting from 0.0.5?  Guess it doesn't really matter.

**Review by ericstj — APPROVED** [review]

**Review by eerhardt — APPROVED** [review]
LGTM. Thanks!

File: Microsoft.ML.TestDatabases.nuspec (hunk 1 above)
Comment (tannergooding) [reply Microsoft.ML.TestDatabases.nuspec:5]: I just copied the existing NuGet.

**Pull request merged**